### ISSUE TITLE
Support arbitrary installation command as source method

### ISSUE THREAD

**Issue by dbkaplun** [jordansissel/fpm#968]
Would be great if I could do something like `fpm -s cmd -t deb curl http://github.com/path/to/install.sh \| sh` or something similar!

**Comment by hatt** [comment]
How would you handle the output of the cmd? Running a cmd is fine but if you don't know what it will actually do, difficult to sandbox and collect... Cool idea though!

**Comment by miked63017** [comment]
What if it was basically the same thing as an empty source, and the command
just ran through pre or post install section? Maybe log to some arbitrary
tmp file during install if needed.

**Comment by dbkaplun** [comment]
Yes, I was thinking something along the lines of @miked63017's comment. The package itself would be completely empty, only containing metadata, and would run a command that does whatever the packager wants. Of course, this would mean that an uninstaller is impossible without a different uninstall command.
